app.js

```diff
@@ -39,6 +39,18 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
+app.get("/", (req, res) => {
+  res.send("Hello World");
+});
+
+app.get("/", (req, res) => {
```

Comment:
I think you are using the same endpoint. Please remove the code or change to something else. `/app-status`